study.md

```diff
@@ -246,5 +254,6 @@ Pick a few design heuristics. Describe some common, named refactorings you can
 use approach a good design.
 
 ```md
-<!-- your answer here -->
+<!-- For instance you could have a method called "make_car" but that would have a lot of code to excute making a car.
+Instead I would brake it up into several function sush as "form_engine" , "paint_exterior" , "mount_seating" etc. -->
```

Comment:
Awesome! This is a great example of a technique called "Extract Method". Have a look at some named refactorings, like "Extract Method" or "Extract Class", to level up.